env_spec.py

```diff
@@ -124,50 +126,57 @@ def parse(env_spec_text):
     comment_regex = r"^(.+)\#(.+)$"
 
     lines = env_spec_text.split("\n")
+    enumerated_list = list(enumerate(lines))
```

Comment:
I think we do not have to wrap `enumerate(...)` in a `list`, right?

Reply:
if I have only enumerated_list = (enumerate(lines)), I will get the enumerated object.

Reply:
Won't the enumerated object work like a list in a `for ... in` loop?

Reply:
Yes, it will.My mistake.

Reply:
Not a problem at all 😄!